packages/components/src/internal/components/base/SelectViewInput.spec.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import ReactSelect from 'react-select';
+import { mount, ReactWrapper } from 'enzyme';
+
+import { SelectView, SelectViewInput, waitForLifecycle } from '../../..';
+import {
+    clearSelectViewsInLocalStorage,
+    getSelectViewsInLocalStorage,
+    setSelectViewInLocalStorage,
+} from './SelectViewInput';
+
+function getDefaultProps() {
+    return {
+        defaultView: SelectView.Grid,
+        id: 'test-select-view-input',
+        onViewSelect: jest.fn(),
+        views: [SelectView.Cards, SelectView.Grid, SelectView.Heatmap],
+    };
+}
+
+function getSelectValue(wrapper: ReactWrapper): any {
+    return wrapper.find('input[name="select-view-input"]').prop('value');
+}
+
+function setSelectValue(wrapper: ReactWrapper, value: any): void {
+    (wrapper.find(ReactSelect).instance() as any).selectValue({ value });
```

Comment:
I saw you had this same line in `AssayTypeSummary.spec.tsx`, any reason to export this as a helper function for that (and other tests) to use?

Reply:
I considered it but given that we're seeking to update `ReactSelect` in the near future I decided to hold off on sharing this to prevent further dependencies at the moment. This helper relies on an internal function of `ReactSelect`, namely `selectValue` that is likely to change or not be there in future versions. Ideally, we'd have a more robust "through the front door" solution to selecting elements in a dropdown or mock out the testing of the dropdown entirely.